File: 485-max-consecutive-ones/max-consecutive-ones.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int ans=0,count=0;
        for(auto it:nums){
            if(it==1){
                count++;
                ans=max(ans,count);
            }else count=0;
        }
        return    ans;
    }
};

File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let ans=0,sum=0;
    for(let i=0;i<nums.length;i++){
        if(nums[i]){
            sum++;
            ans=Math.max(ans,sum);
        }else sum=0;
    }
    return ans;
};